File: 3830-find-closest-person/find-closest-person.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        disx = abs(z-x)
        disy = abs(z-y)
        if disx < disy:
            return 1
        elif disx == disy:
            return 0
        else: 
            return 2
        

File: 3830-find-closest-person/find-closest-person.java
class Solution {
    public int findClosest(int x, int y, int z) {
        int disx = Math.abs(z-x);
        int disy = Math.abs(z-y);
        return disx < disy ? 1 : disx == disy ? 0 : 2;
    }
}